common/src/app/common/files/changes_builder.cljc

```diff
@@ -742,13 +742,14 @@
       changes)))
 
 (defn delete-component
-  [changes id]
+  [changes id page-id]
   (assert-library! changes)
   (-> changes
       (update :redo-changes conj {:type :del-component
                                   :id id})
       (update :undo-changes conj {:type :restore-component
-                                  :id id})))
+                                  :id id
+                                  :page-id page-id})))
```

Comment:
I think this param should be added to the change schema on app.common.files.changes